src/Components/Clock.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+type State = {
+  time: Date;
```

Comment:
How can a Date be a time type?)
```suggestion
  currentDate: Date;
```